### PR TITLE
tests(azure-storage-service): fix test

### PR DESCRIPTION
**Pull request by Tony133**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/nestjs/nest/blob/master/CONTRIBUTING.md
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[x] Other... Please describe:
```

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

## Other information
This PR, fix test error "should fail upload when File is null"

### CODE REVIEW

**Comment by Tony133** [comment]
to be evaluated :-)

**Comment by manekinekko** [comment]
Thank you for this contribution 👍

**Comment by manekinekko** [comment]
@Tony133  Could you provide more feedback about the reason for this change?

**Comment by Tony133** [comment]
Hi @manekinekko, I made this change because I had made another pull this https://github.com/nestjs/azure-storage/pull/144

gave this error in tests: 

```PASS  lib/azure-nest-storage.module.spec.ts
 FAIL  lib/azure-storage.service.spec.ts
  ● AzureStorageService › should fail upload when File is null

    expect(received).toBe(expected) // Object.is equality

    Expected: "TypeError: Cannot destructure property `buffer` of 'undefined' or 'null'."
    Received: "TypeError: Cannot destructure property 'buffer' of 'file' as it is null."

      123 |       await storage.upload(null);
      124 |     } catch (e) {
    > 125 |       expect(e.toString()).toBe(
          |                            ^
      126 |         "TypeError: Cannot destructure property `buffer` of 'undefined' or 'null'.",
      127 |       );
      128 |     }
```

which has nothing to do with what I had done, so I tried to see if I could solve it.

The solution at the moment looks like this I hope it will go well.

File: lib/azure-storage.service.spec.ts [hunk 1]
```diff
@@ -120,7 +120,7 @@ describe('AzureStorageService', () => {
   });
   it('should fail upload when File is null', async () => {
     try {
-      await storage.upload(null);
+      await storage.upload(file, null);
```
Comment (manekinekko) [inline lib/azure-storage.service.spec.ts:123]:
```suggestion
      await storage.upload(null);
```

File: lib/azure-storage.service.spec.ts [hunk 2]
```diff
@@ -120,7 +120,7 @@ describe('AzureStorageService', () => {
...
     } catch (e) {
       expect(e.toString()).toBe(
         "TypeError: Cannot destructure property `buffer` of 'undefined' or 'null'.",
```
Comment (manekinekko) [inline lib/azure-storage.service.spec.ts:126]:
```suggestion
        "TypeError: Cannot destructure property 'buffer' of 'file' as it is null."
```

**Review by manekinekko — CHANGES_REQUESTED** [review]

**Comment by manekinekko** [comment]
Oh! I see what was the error.

It seems that the error message of the exception that is thrown when a file is `null`, has changed:

from:
```
TypeError: Cannot destructure property `buffer` of 'undefined' or 'null'.
```

To:
```
TypeError: Cannot destructure property 'buffer' of 'file' as it is null.
```

I've sent you few changes to make before merging the PR.

File: lib/azure-storage.service.spec.ts (hunk 1 above)
Comment (Tony133) [reply lib/azure-storage.service.spec.ts:123]: So does this part come back as before?

**Comment by jasonpaige** [comment]
Any chance this can get merged so that the build can be fixed please :pray: ? I'm keen to make use of the feature added with https://github.com/nestjs/azure-storage/pull/65 but the build has been broken since it was merged in last year :(

**Pull request merged**